run-e2e.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash -xe
 rc=0
-docker-compose -f ./docker-compose.core.yml up -d --force-recreate
-docker-compose -f ./docker-compose.core.yml exec web ./scripts/setup
-docker-compose -f ./docker-compose.core.yml exec web-next inspirehep db create
-docker-compose -f ./docker-compose.core.yml exec web inspirehep importer records -d data/records/authors
-docker-compose -f ./docker-compose.core.yml exec web inspirehep importer records -f data/records/conference/7577512.json
-docker-compose -f ./docker-compose.core.yml -f ./e2e/docker-compose.e2e.yml run --rm e2e bash -c "cd /opt/e2e && yarn && yarn test" || rc=$?
+docker-compose up -d --force-recreate
+docker-compose exec web ./scripts/setup
+docker-compose exec web-next inspirehep db create
+docker-compose exec web inspirehep importer records -d data/records/authors
+docker-compose exec web inspirehep importer records -f data/records/conference/7577512.json
+docker-compose -f ./docker-compose.yml -f ./docker-compose.override.yml -f ./e2e/docker-compose.e2e.yml run --rm e2e bash -c "cd /opt/e2e && yarn && yarn test" || rc=$?
```

Comment:
IIUC, `docker-compose.core.yml`->`docker-compose.yml` and `docker-compose.dev.yml` -> `docker-compose.override.yml`. So here it looks like you're adding the `override` file which was not there previously. Is that on purpose?

Reply:
Yes, override file allows to override config from docker-compose file https://docs.docker.com/compose/extends/#understanding-multiple-compose-files 
Thanks to this change we don't have to use ./docker-inspier script anymore. 
Here I'm providing override as in e2e tests I would like to use quicker build, but If you think that we should use full build then it can be removed.

Reply:
Ok, I noticed that running e2e with dev dockerfile is not working unfortunately. Restoring to previous e2e setup.